src/Multiselect/MultiselectComponents.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import classNames from 'classnames';
+import { components } from 'react-select';
```

Comment:
nit: `import { components as reactSelectComponents } from 'react-select';` just to be a tad more explicit